home/import_assessments.py

```diff
@@ -403,6 +415,33 @@ def get_content_page_id_from_slug(slug: str) -> int:
     return page.id
 
 
+def check_punctuation(
+    high_inflection: str | None, medium_inflection: str | None
+) -> None:
+    if high_inflection is not None:
+        try:
+            high_inflection.isdigit()
+        except AttributeError:
+            high_inflection = str(high_inflection)
+        punctuaton = bool(re.search(r",", high_inflection))
+        if punctuaton:
+            raise ImportAssessmentException(
+                "Invalid number format for high inflection. "
+                "Please use '.' instead of ',' for decimals."
+            )
```

Comment:
I think we can also add the row number to this exception (there's a row_num parameter for `ImportAssessmentException`)

Reply:
Cool